tests/unit/entries/test_soundcloud_entries.py

```diff
@@ -23,6 +28,11 @@ def is_premiere():
     return False
 
 
+@pytest.fixture()
+def track_count():
+    return 1
```

Comment:
dupe fixtures

Reply:
after these two small fixes it should be good to merge

Reply:
Fixed those two problems, thanks for being patient this has been quite the pull request at least it has been enjoyable.

Reply:
No problem, thank _you_ for contributing! I hope my comments have been helpful and that you find this app useful